src/owasp-dependency-check-false-positives.xml

```diff
@@ -59,4 +59,68 @@
     <packageUrl regex="true">^pkg:maven/io\.netty/netty\-tcnative\-classes@.*$</packageUrl>
     <cpe>cpe:/a:netty:netty</cpe>
   </suppress>
+  <suppress>
+    <notes><![CDATA[
+   file name: zookeeper-3.8.0.jar
```

Comment:
please add a meaningful explanation.
Some thing like "This is a false positive, the CVE is not about Apache ZooKeeper"

Reply:
you can create a single entry with multiple CVEs
this way it is simpler to maintain the list, as all those exclusions are due to the same false positive